akka-persistence/src/main/scala/akka/persistence/journal/leveldb/LeveldbJournal.scala

```diff
@@ -43,18 +71,31 @@ private[persistence] object LeveldbJournal {
    * Used by query-side. The journal will send [[EventAppended]] messages to
    * the subscriber when `asyncWriteMessages` has been called.
    */
-  case class SubscribePersistenceId(persistenceId: String) extends SubscriptionCommand
-  case class EventAppended(persistenceId: String) extends DeadLetterSuppression
+  final case class SubscribePersistenceId(persistenceId: String) extends SubscriptionCommand
+  final case class EventAppended(persistenceId: String) extends DeadLetterSuppression
 
   /**
-   * Subscribe the `sender` to changes (appended events) for a specific `persistenceId`.
+   * Subscribe the `sender` to current and new persistenceIds.
    * Used by query-side. The journal will send one [[CurrentPersistenceIds]] to the
    * subscriber followed by [[PersistenceIdAdded]] messages when new persistenceIds
    * are created.
    */
-  case object SubscribeAllPersistenceIds extends SubscriptionCommand
-  case class CurrentPersistenceIds(allPersistenceIds: Set[String]) extends DeadLetterSuppression
-  case class PersistenceIdAdded(persistenceId: String) extends DeadLetterSuppression
+  final case object SubscribeAllPersistenceIds extends SubscriptionCommand
+  final case class CurrentPersistenceIds(allPersistenceIds: Set[String]) extends DeadLetterSuppression
+  final case class PersistenceIdAdded(persistenceId: String) extends DeadLetterSuppression
+
+  /**
+   * Subscribe the `sender` to changes (appended events) for a specific `tag`.
```

Comment:
Mention the `Tagged` here? So people know how to "tag"?
Some docs will need to explain it anyway, tomorrow though I guess?